File: FrontEnd/MedicApp/DTO's/PatientDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MedicApp.DTO_s
{
    public class PatientDTO
    {
        public long id { get; set; }
        public string name { get; set; }
        public string lastName { get; set; }
        public string department { get; set; }
        public int gender { get; set; }
        public float weight { get; set; }
        public float size { get; set; }
        public int year { get; set; }
        public string observation { get; set; }

        public int status { get; set; }

        public string StatusText => status != 1 ? "Inactivo" : "Activo";

        public string StatusColor => status != 1 ? "Red" : "Green";
    }
}

File: src/main/java/com/uisrael/medical_service/dtos/PatientDTO.java
package com.uisrael.medical_service.dtos;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.UUID;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class PatientDTO {
    private UUID idPatient;
    private String name;
    private String lastName;
    private String department;
    private Integer gender;
    private Float weight;
    private Float size;
    private Integer year;
    private String observation;
    private Integer status;

}
